rocky/rocky/views/scheduler.py

```diff
@@ -0,0 +1,190 @@
+from datetime import datetime
+from typing import Any
+
+from django.contrib import messages
+from django.http import JsonResponse
+from django.utils.translation import gettext_lazy as _
+from katalogus.client import Boefje, Normalizer
+from tools.forms.scheduler import TaskFilterForm
+
+from octopoes.models import OOI
+from rocky.scheduler import Boefje as SchedulerBoefje
+from rocky.scheduler import (
+    BoefjeTask,
+    LazyTaskList,
+    NormalizerTask,
+    PrioritizedItem,
+    RawData,
+    SchedulerError,
+    Task,
+    scheduler_client,
+)
+from rocky.scheduler import Normalizer as SchedulerNormalizer
+from rocky.views.mixins import OctopoesView
+
+
+def get_date_time(date: str | None) -> datetime | None:
+    if date:
+        return datetime.strptime(date, "%Y-%m-%d")
+    return None
+
+
+class SchedulerView(OctopoesView):
+    task_type: str = "boefje"  # default task type
+    task_filter_form = TaskFilterForm
+
+    def setup(self, request, *args, **kwargs):
+        super().setup(request, *args, **kwargs)
+        self.scheduler_client = scheduler_client(self.organization.code)
+
+    def get_task_filters(self) -> dict[str, Any]:
+        return {
+            "scheduler_id": f"{self.task_type}-{self.organization.code}",
+            "task_type": self.request.GET.get("type", self.task_type),
```

Comment:
Should self.task_type not be used on line 43 too, or should we not read from the GET request for both?

Reply:
removed `self.request.GET.get` completely